docs/node-operators/network_restarts.md

```diff
@@ -0,0 +1,170 @@
+    AutoCertDomain = ""
+    CertificateFile = ""
+    KeyFile = ""                        # to there
+  [Gateway.REST]
+    Port = 3009
+    IP = "0.0.0.0"
+    Enabled = true
+    APMEnabled = true
+```
+
+If you do not want to enable ssl make sure to set `HTTPSEnabled` to false or the data-node would not start properly.
+
+### Step 1: Stop the network
+
+Wait for a new checkpoint file to be produce then stop all the nodes of the network (vega, data-node and tendermint). Then backup all tendermint chain data and vega data.
+Save the selected checkpoint file in a safe location you will need to reuse it later.
+
+:::info
+You can locate all your node checkpoint under: `/path/to/vega/home/vega/node/checkpoints`
+You also can get a list of all path used by your node using `vega paths list`, the checkpoints folder path is `CheckpointStateHome` under this list.
+:::
+
+You can now remove all previous state of the chain by doing:
+```
+vega --home=/path/to/vega/home unsafe_reset_all
+vega tm --home=/path/to/tendermint/home unsafe_reset_all
+rm -rf /path/to/data-node/home/vega/data-node/storage/
+```
+
+:::info
+The exact path of the data-node folder to remove can be found using `vega paths list`, this path is `DataNodeStorageHome` under this list.
+:::
+
+### Step 2: Update the genesis file
+
+One of the validators will now need to update the [genesis file](https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json) with the following information:
+- the new start date of the network
+- the new network ID
+- the hash of the checkpoint file to be used.
+
+This should be done via a pull request on the [networks](https://github.com/vegaprotocol/networks) repo and ideally approved by 2/3+1 of all validators.
+
+### Step 3: Restart the network
+
+:::note
+This is a critical part which needs to be done with all validator synchronously (this will not be needed in future updates).
+:::
+
+All validators needs to restart their node in a synchronous way. The reason is that the network needs to synchronise it's state with the ethereum state of vega token delegation to the vega network. This is done during the bootstrapping period which happend during the N first block of the chain (this can be configured in the genesis file).
+
+During the bootstraping no transaction from users can be emitted apart from the transaction to submit the checkpoint. This should be done only once by one of the validators using the following command:
+```
+vega checkpoint restore --home=/path/to/vega/home --passphrase-file=YOUR_NODEWALLET_PASSPHRASE_FILE --checkpoint-file=/path/the/
+checkpoint/file
+```
+
+Once this is done, we will need to monitor the network to make sure all delegation are recovered properly by the end of the bootstraping period
+
+### Misc
+
+The ethereum event forwarder is now not required anymore, this should be removed from your infrastructure and not started. This service have been re-written and is not integrated in the vega core node.
```

Comment:
```suggestion
The ethereum event forwarder is now not required anymore, this should be removed from your infrastructure and not started. This service has been re-written and is now integrated in the vega core node.
```

Reply:
Most comments are minor, however @jeremyletang - please confirm on this that my suggested change of the word `not` to `now` is correct. Thanks

Reply:
You are correct gordon. Thanks.